### DISCUSSION TITLE
New Dark themes for password manager mobile apps.

### DISCUSSION THREAD

**Discussion by KeshavPratap-K** [bitwarden/clients#5921] [Password Manager]
### Code Contribution Proposal

Two need dark themes are added to give Bitwarden mobile app a refreshed look. One named DarkPlus inspired from GitHub dark mode and Solarized DarkBlue is inspired from existing Solarized Dark theme to be in blue accent color.

## Screenshots

<img src="https://github.com/bitwarden/mobile/assets/47635515/c2fb54e7-e66c-4b3c-9a7e-ee9c39f4a208" height="600">
<p align = "left">
<b>Dark+ LoginPage</b>
</p>
<img src="https://github.com/bitwarden/mobile/assets/47635515/bb8ac6de-b6ae-4522-bbb1-e2f097f048cd" height="600">
<p align = "left">
<b>Solarized DarkBlue LoginPage</b>
</p>

<img src="https://github.com/bitwarden/mobile/assets/47635515/264f4cd0-6c45-404f-bc5c-2defc2b45f8b" height="600">
<p align = "left">
<b>Dark+ Password</b>
</p>
<img src="https://github.com/bitwarden/mobile/assets/47635515/2d357d2c-0b33-44cb-bdc3-f3ec8a9dd1fe" height="600">
<p align = "left">
<b>Solarized DarkBlue Password</b>
</p>

<img src="https://github.com/bitwarden/mobile/assets/47635515/328329ce-867e-4ee9-b021-e8a1e2a7dbb8" height="600">
<p align = "left">
<b>Dark+ Password generator</b>
</p>
<img src="https://github.com/bitwarden/mobile/assets/47635515/48963cd2-80dc-4226-bdae-440dbf42142c" height="600">
<p align = "left">
<b>[olarized DarkBlue Password generator</b>
</p>

<img src="https://github.com/bitwarden/mobile/assets/47635515/74aaecc3-3358-4a84-9a59-29f49bfaef2e" height="600">
<p align = "left">
<b>Dark+ HomePage</b>
</p>
<img src="https://github.com/bitwarden/mobile/assets/47635515/0685c66b-3756-4dce-9294-9a87e61de14f" height="600">
<p align = "left">
<b>Solarized DarkBlue HomePage</b>
</p>

<img src="https://github.com/bitwarden/mobile/assets/47635515/2d5ef9cd-dda5-492a-b94e-6579b5b8be4f" height="600">
<p align = "left">
<b>Options theme Value Dark+</b>
</p>
<img src="https://github.com/bitwarden/mobile/assets/47635515/6117f70f-7895-49bd-9aac-037fe59e6ddb" height="600">
<p align = "left">
<b>Options theme Value Solarized DarkBlue</b>
</p>

<img src="https://github.com/bitwarden/mobile/assets/47635515/064a0824-cf2a-4ab3-981f-9ae335bb5b40" height="600">
<p align = "left">
<b>Settings</b>
</p>

**Comment by KeshavPratap-K** [comment]
Adding code change pull request link - https://github.com/bitwarden/mobile/pull/2605

**Reply by dwbit** [reply]
Hey @KeshavPratap-K thanks for the submission, the team will review and get back to you shortly!

**Comment by dwbit** [comment]
Hey @KeshavPratap-K, the team has confirmed that they aren't currently accepting new themes, but they are investigating the possibility of user customization for themes in the future. As such, I'll close this one out for now.
